docs/workflows/sync-collection.rst

```diff
@@ -28,12 +28,12 @@ Reference (pulpcore): `Repository API Usage
 Create a CollectionRemote
 -------------------------
 
-Creating a CollectionRemote object allows Pulp to sync a whitelist of Collections from an external
+Creating a CollectionRemote object allows Pulp to sync Collections from an external
 content source. This is most commonly is ``https://galaxy.ansible.com/`` or another ``pulp_ansible``
 instance.
 
-In this example we will be syncing the ``testing.ansible_testing_content`` Collection from
-``https://galaxy-dev.ansible.com``.
+In this example we will be syncing the Collection with ``namespace=testing``  and ``name=ansible_testing_content``
```

Comment:
Can this identify to the user that all versions will be sync'd. It reads currently like only 1 will, but in practice it will be all verisons of this namespace and name.